src/puppetlabs/trapperkeeper/services/webserver/jetty9_core.clj

```diff
@@ -702,8 +704,9 @@
          enable-trailing-slash-redirect? (:enable-trailing-slash-redirect? options)
          normalize-request-uri? (:normalize-request-uri? options)]
      (.setBaseResource handler (Resource/newResource base-path))
-     (when follow-links?
-       (.addAliasCheck handler (AllowSymLinkAliasChecker.)))
+     (if follow-links?
+       (.addAliasCheck handler (AllowSymLinkAliasChecker.))
```

Comment:
Would it be better here to call `.setAliasChecks` with a list containing just `AllowSymLinkAliasChecker`?  Wasn't sure what would happen if you end up adding more than one instance of the same alias checker to the list - like for the case where there's an `AllowSymLinkAliasChecker` that's already been added to the handler by default.